API.apib

```diff
@@ -234,3 +234,30 @@ Response 200 (application/json)
             ]
      }
   }
+
+# Get path from user location to specific store
+GET api/v1/tch/stores/:store_id/path{?address}
+ Parameters:
+     + store_id (UUID)
```

Comment:
should add optional or required for each param, to ensure FE validation for correct one